docs/authorization/globus_app/apps.rst

```diff
@@ -0,0 +1,89 @@
+.. _globus_apps:
+
+.. currentmodule:: globus_sdk
+
+GlobusApps
+==========
+
+
+.. warning::
+
+    Currently ``GlobusApp`` only supports scripting-based use cases.
```

Comment:
Workshopping...

```suggestion
    Currently, ``GlobusApp`` can be used in scripts (e.g., notebooks or automation) and local applications launched directly by a user.
```

Reply:
If we're making notes on this warning, I would like to see it converted from a `warning` to a `note` to be less of an attention-grab at the start of the doc.

I might phrase it as
> ``GlobusApp`` only supports client applications, where a user or client logs in or authenticates.
> Resource server implementations should not use ``GlobusApp``.

This phrasing reveals my belief that any abstraction aimed at Resource Server implementations will be fully distinct from ``GlobusApp``.

Reply:
Reclassified as a note & updated the wording here; I pulled from both of your suggestions for the final copy.